code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -0,0 +1,267 @@
+	adjustFireLoss(-maxHealth/2)
+	adjustToxLoss(-maxHealth/2)
+	adjustOxyLoss(-maxHealth/2)
+
+/mob/living/simple_animal/hostile/eldritch/armsy/AttackingTarget()
+	if(istype(target,/obj/item/bodypart/r_arm) || istype(target,/obj/item/bodypart/l_arm))
+		target.Destroy()
+		heal()
+		return
+	if(back)
+		back.target = target
+		back.AttackingTarget()
+	if(!Adjacent(target))
+		return
+	do_attack_animation(target)
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		if(HAS_TRAIT(C, TRAIT_NODISMEMBER))
+			return
+		var/list/parts = list()
+		for(var/X in C.bodyparts)
+			var/obj/item/bodypart/BP = X
+			if(BP.body_part != HEAD && BP.body_part != CHEST && BP.body_part != LEG_LEFT && BP.body_part != LEG_RIGHT)
+				if(BP.dismemberable)
+					parts += BP
+		if(prob(10))
+			var/obj/item/bodypart/BP = pick(parts)
+			if(BP)
+				BP.dismember()
+
+	. = ..()
+
+/mob/living/simple_animal/hostile/eldritch/armsy/prime
+	name = "Lord of the Night"
+	real_name = "Master of Decay"
+	maxHealth = 400
+	health = 400
+	melee_damage_lower = 20
+	melee_damage_upper = 25
+
+/mob/living/simple_animal/hostile/eldritch/armsy/prime/New(spawn_more, len)
+	var/matrix/M = matrix()
+	M.Scale(1.4,1.4)
+	src.transform = M
+	. = ..()
+
+/mob/living/simple_animal/hostile/eldritch/rust_spirit
+	name = "Rust Walker"
+	real_name = "Rusty"
+	desc = "Incomprehensible abomination actively seeping life out of it's surrounding."
+	icon_state = "rust_walker"
+	status_flags = CANPUSH
+	icon_living = "rust_walker"
+	maxHealth = 50
+	health = 50
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	sight = SEE_TURFS
+
+/mob/living/simple_animal/hostile/eldritch/rust_spirit/Initialize()
+	. = ..()
+	AddSpell(/obj/effect/proc_holder/spell/aoe_turf/rust_conversion/small)
+	AddSpell(/obj/effect/proc_holder/spell/targeted/projectile/dumbfire/rust_wave/short)
+
+/mob/living/simple_animal/hostile/eldritch/ash_spirit
+	name = "Ash Man"
+	real_name = "Ashy"
+	desc = "Incomprehensible abomination actively seeping life out of it's surrounding."
+	icon_state = "ash_walker"
+	status_flags = CANPUSH
+	icon_living = "ash_walker"
+	maxHealth = 50
+	health = 50
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	sight = SEE_TURFS
+
+/mob/living/simple_animal/hostile/eldritch/ash_spirit/Initialize()
+	. = ..()
+	AddSpell(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash)
+	AddSpell(/obj/effect/proc_holder/spell/pointed/ash_cleave/long)
+
+/mob/living/simple_animal/hostile/eldritch/stalker
+	name = "Flesh Stalker"
+	real_name = "Flesh Stalker"
+	desc = "Abomination made from severed limbs."
+	icon_state = "stalker"
+	status_flags = CANPUSH
+	icon_living = "stalker"
+	maxHealth = 150
+	health = 150
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	sight = SEE_MOBS
+
+/mob/living/simple_animal/hostile/eldritch/stalker/Initialize()
+	AddSpell(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash)
+	AddSpell(/obj/effect/proc_holder/spell/targeted/shapeshift)
```

Comment:
me and the boys farming chaos magicarp bolts until one of us turns into a headslug:
https://giphy.com/gifs/PvDhtYE7qDftS

Reply:
also, holy FUCK do these guys blow ash men out of the water

3x as much health, AND they can jaunt, AND they can open doors, AND they can one click stun people by turning into a beepsky

Reply:
I actually really love them as a concept, though

give them a subtype of the shapeshift spell that can only turn them into "harmless" animals (y'know, pets, farm animals, etc.) instead of the wizard shapeshift spell they have now, drop the ethereal jaunt ability, move the knock ability to ash men (and seriously buff the stats of ash men and/or give them ninja invisibility or something), and maybe cut the health totals of these guys down a bit (to, say, 100?). ooh, and give these guys something like that tracking "spell" from that transcendent olfaction mutation (or, like, innate access to a suit sensors menu and an innate crew pinpointer ability, sort of like an upgraded mediborg), so that they can track people down.

the roles of these guys would then be to turn into cuddly animals, track down targets while in animal form, then turn into a horrific eldritch monstrosity right next to said target and start mauling them. if they've been outed, they can fall back on ventcrawling over to their target, then popping out of a vent, returning to their "true form", and mauling them that way.

Reply:
i will give them an emp spell instead of knock